dockerfiles/mlrun-api/Dockerfile

```
@@ -51,14 +51,28 @@ RUN python -m pip install --upgrade pip~=${MLRUN_PIP_VERSION}
 
 WORKDIR /mlrun
 
-COPY ./dockerfiles/mlrun-api/requirements.txt ./mlrun-api-requirements.txt
+COPY ./dockerfiles/mlrun-api/requirements.txt dockerfiles/mlrun-api/requirements.txt
 COPY ./extras-requirements.txt ./extras-requirements.txt
 COPY ./requirements.txt ./
+COPY ./dockerfiles ./dockerfiles
+COPY ./pipeline-adapters ./pipeline-adapters
+COPY ./requirements.txt  .
+COPY ./dev-requirements.txt  .
+COPY ./conda-arm64-requirements.txt  .
+COPY ./packages.py .
+COPY ./dependencies.py  .
+COPY ./setup.py .
+COPY ./README.md .
+COPY ./mlrun/utils/version/version.json ./mlrun/utils/version/version.json
 
 RUN python -m pip install \
     -r requirements.txt \
     -r extras-requirements.txt \
-    -r mlrun-api-requirements.txt
+    -r dockerfiles/mlrun-api/requirements.txt  --no-cache-dir  --no-build-isolation && python -m pip install .[complete-api] --no-cache-dir --no-build-isolation &&\
```

```suggestion
    -r dockerfiles/mlrun-api/requirements.txt  --no-cache-dir  --no-build-isolation && \
    python -m pip install .[complete-api] --no-cache-dir --no-build-isolation && \
```
Use clear line brekass after `&&`
```suggestion
    -r dockerfiles/mlrun-api/requirements.txt  --no-cache-dir  --no-build-isolation && \
    python -m pip install .[complete-api] --no-cache-dir --no-build-isolation && \
```